### DISCUSSION TITLE
Audit cycle timeout, please pay attention to thank you!

### DISCUSSION THREAD

**Discussion by Mr-LiXiantong** [microsoft/MicrosoftEdge-Extensions#113] [Q&A]
I initiated the plug-in on 18th, why it has not been reviewed for 9 days, shouldn't I receive the relevant notice of review within 7 days? Thank you for your help and verification from relevant Edge staff. Thanks for your hard work! Commodity 
identification：0RDCKD0TT8WB ，Please help deal with the release situation. Thank you！

**Comment by Bhuvana-Priya** [comment]
Hi @lixiantongll,

We apologize for the delay and our team is working on improving the review timelines. Requesting your patience in the meanwhile and do let us know if this continues for the next reviews.
